profiles/common/modules/features/multisite_business_indicators/multisite_business_indicators_standard/includes/views_handler_field_ecas_user_roles.inc

```diff
@@ -11,17 +11,27 @@
  * @ingroup views_field_handlers
  */
 class views_handler_field_ecas_user_roles extends views_handler_field_prerender_list {
-  function construct() {
+
+  /**
+   * Class constructor.
+   */
+  public function construct() {
```

Comment:
Lol a fake constructor? Views surprises me sometimes :)

This is overriding a method on the parent class, so the proper documentation is `{@inheritdoc}`.